themes/src/main/resources/theme/base/login/login-otp.ftl

```diff
@@ -51,16 +51,16 @@
             <script type="text/javascript">
             $(document).ready(function() {
                 // Card Single Select
-                $('.card-pf-view-single-select').click(function() {
-                  if ($(this).hasClass('active'))
-                  { $(this).removeClass('active'); $(this).children().removeAttr('name'); }
+                $('.otp-tile').click(function() {
```

Comment:
This is a bit off topic for this PR as we used JS here as well (but I just noticed it now :)). Didn't we have some policy not to use any JS on login pages where possible (of course e.g. webauthn wouldn't work without JS). @ssilvert?

Reply:
As you pointed, we have JS already, so no regression of this PR :-) But we have follow-up JIRA for remove JS from login pages.

Reply:
Ah, sorry – I forgot about [KEYCLOAK-12823](https://issues.redhat.com/browse/KEYCLOAK-12823) (even though I was the one who created it :)). On the other hand, that JIRA covers the authenticator selector page, not OTP page. Or did you mean some other JIRA.

Reply:
Nope, I meant this one and forgot that it is specific only to authenticator selector page :-) Maybe we can slightly rename the existing JIRA and/or update it's description to cover both screens? Or create new JIRA?

Reply:
I've updated KEYCLOAK-12823.